benefits/static/css/styles.css

```diff
@@ -517,26 +533,18 @@ footer .footer-links li a.footer-link:visited {
 }
 
 /* Forms: Text Inputs */
-/* Form Container: Must use .form-container parent to use these styles */
+/* Form Field Container: Must use .form-container parent to use these styles */
```

Comment:
Renamed this for clarity. Each `.form-field-container` wraps one of these as a unit
<img width="665" alt="image" src="https://github.com/user-attachments/assets/1a28923b-88a5-4b78-a79f-874bb7bffd6d" />

`.form-container` made it seem like it was wrapping the entire form, which is kinda true for Elig Index but not on Elig Confirm.